install/0000_90_machine-config_01_node_dashboard.yaml

```diff
@@ -2793,7 +2791,7 @@ metadata:
     include.release.openshift.io/self-managed-high-availability: "true"
     include.release.openshift.io/single-node-developer: "true"
   labels:
-    app.kubernetes.io/part-of: openshift-monitoring
+    app.kubernetes.io/part-of: openshift-machine-config-operator
```

Comment:
maybe use `app.kubernetes.io/managed-by` instead?
(app.kubernetes.io/managed-by for the operator and app.kubernetes.io/part-of for the stack?)

Reply:
(we could use `app.kubernetes.io/managed-by: cluster-version-operator` but others don't have it)
/lgtm